webapp/app.py

```diff
@@ -70,7 +70,11 @@ def examples_view():
             }
         )
 
-    return flask.render_template("examples/index.html", examples=examples)
+    show_standalone = "standalone" in flask.request.args
```

Comment:
I'm not crazy about this idea. If we're fundamentally showing a different page for standalone examples, why wouldn't it be at a different URL? (e.g. `/standalone-examples`). Which then has implications for the path for the standalone examples themselves. In any case, how were you hoping to link to the standalone version?

But I think it's probably neater to show all the URLs on the same page, or add a client-side switch on the page or something.